test/NuGet.Core.FuncTests/NuGet.Signing.CrossFramework.Test/CrossFrameworkVerificationTest.cs

```diff
@@ -0,0 +1,425 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#if IS_SIGNING_SUPPORTED
+
+using System;
+using System.Security.Cryptography.X509Certificates;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NuGet.Test.Utility;
+using Test.Utility.Signing;
+using Xunit;
+
+namespace NuGet.Signing.CrossFramework.Test
+{
+    [Collection(CrossVerifyTestCollection.Name)]
+    public class CrossFrameworkVerificationTest
+    {
+        private const int SHA1HashLength = 20;
+
+        private readonly string _successfullyVerified = "Successfully verified package 'packageA.1.0.0'";
+        private readonly string _noTimestamperWarning = "NU3027: The signature should be timestamped to enable long-term signature validity after the certificate has expired";
+        private readonly CrossVerifyTestFixture _testFixture;
+
+        public CrossFrameworkVerificationTest(CrossVerifyTestFixture fixture)
+        {
+            _testFixture = fixture ?? throw new ArgumentNullException(nameof(fixture));
+        }
+
+        [Fact]
+        public async Task Verify_AuthorSignedPackage_SuccessAsync()
+        {
+            // Arrange
+            var nupkg = new SimpleTestPackageContext();
```

Comment:
No, it doesn't create nupkg file on disk yet.
The nupkg file is created in [this line](https://github.com/NuGet/NuGet.Client/blob/dev-hengliu-xplat4-cross-verify/test/NuGet.Core.FuncTests/NuGet.Signing.CrossFramework.Test/CrossFrameworkVerificationTest.cs#L42).
And the details are [here](https://github.com/NuGet/NuGet.Client/blob/dev/test/TestUtilities/Test.Utility/Signing/SignedArchiveTestUtility.cs#L45-L52).
Since the signed package is under `dir`, when the test finishes, the TestDirectory will call the [Dispose() method](https://github.com/NuGet/NuGet.Client/blob/dev/test/TestUtilities/Test.Utility/TestDirectory.cs#L28-L37) and delete everything under the TestDirectory, including the signed nupkg file from disk.